geometry_msgs/Point[] corners
bool is_pose_estimated
geometry_msgs/PoseStamped pose
int32 id
float32 tag_size
